examples/website-demos/social-messaging/src/components/Message/SocialMessageUI.tsx

```diff
@@ -0,0 +1,120 @@
+import {
+  Avatar,
+  MessageText,
+  MessageTimestamp,
+  MessageUIComponentProps,
+  useChannelStateContext,
+  useChatContext,
+  useMessageContext,
+} from 'stream-chat-react';
+import { DeliveredCheckmark, DoubleCheckmark } from '../../assets';
+
+import {
+  SocialAttachmentType,
+  SocialChannelType,
+  SocialCommandType,
+  SocialEventType,
+  SocialMessageType,
+  SocialReactionType,
+  SocialUserType,
+} from '../ChatContainer/ChatContainer';
+
+import './SocialMessageUI.scss';
+
+export const SocialMessage: React.FC<
+  MessageUIComponentProps<
+    SocialAttachmentType,
+    SocialChannelType,
+    SocialCommandType,
+    SocialEventType,
+    SocialMessageType,
+    SocialReactionType,
+    SocialUserType
+  >
+> = (props) => {
+  const { channel } = useChannelStateContext<
+    SocialAttachmentType,
+    SocialChannelType,
+    SocialCommandType,
+    SocialEventType,
+    SocialMessageType,
+    SocialReactionType,
+    SocialUserType
+  >();
+  const { client } = useChatContext<
+    SocialAttachmentType,
+    SocialChannelType,
+    SocialCommandType,
+    SocialEventType,
+    SocialMessageType,
+    SocialReactionType,
+    SocialUserType
```

Comment:
do channel and client need generics here? i see we're using `message.user.image` which i assume is a custom type, so that one is needed, but not sure we need on the other two